js/interface.js

```diff
@@ -1968,12 +2048,23 @@ $allowBtnFilter.click(function(event) {
   event.preventDefault();
 
   var $usersFilter = $('.users-filter');
+  var $specificTokens = $('.tokens-list');
   var value = $(this).data('allow');
 
   $allowBtnFilter.removeClass('selected');
   $(this).addClass('selected');
 
   $usersFilter.toggleClass('hidden', value !== 'filter');
+  $specificTokens.toggleClass('hidden', value !== 'tokens');
+
+  if (value === 'tokens') {
+    var tpl = Fliplet.Widget.Templates['templates.apiTokenList'];
+
+    $('.tokens-list').html(tpl({
+      integrationTokenList: integrationTokenList
+    }));
+    $(".tokens-list option[value='" + widgetData.tokenId + "']").attr('selected', 'selected');
```

Comment:
```suggestion
    $(".tokens-list option[value='" + widgetData.tokenId + "']").prop('selected', true);
```

Reply:
Did the changes as per your suggestion and used the `prop` property here.